src/core/public/_variables.scss

```diff
@@ -1,3 +1,6 @@
 @import "@elastic/eui/src/global_styling/variables/header";
 
 $osdHeaderOffset: $euiHeaderHeightCompensation;
+$osdPacificSky: #b9d9eb;
+$osdSanFranciscoFog: #d9e1e2;
+$osdMidnightSky: #002a3a;
```

Comment:
These names reference the actual color which makes me ask if we would change the variable name when we change the color. The names should be related to their purpose.

Reply:
let me fix that